src/main/kotlin/com/ampnet/web3provider/controller/pojo/JsonRpcRequest.kt

```diff
@@ -0,0 +1,8 @@
+package com.ampnet.web3provider.controller.pojo
+
+data class JsonRpcRequest(
```

Comment:
I wouldn't set default values in requests, especially if you forward those requests to other providers.